projects/libs/core-library-typescript/test/gossip/gossip.spec.ts

```diff
@@ -46,7 +45,7 @@ describe("Gossip", function () {
         const consensus = consensuses[i];
         if (i == recipientId) {
           expect(consensus.gossipMessageReceived).to.have.been.calledOnce;
-          expect(consensus.gossipMessageReceived.getCall(0).args[0]).to.have.property("buffer").which.equalBytes(buffer);
+          expect((<sinon.SinonSpy>consensus.gossipMessageReceived).getCall(0).args[0]).to.have.property("buffer").which.deep.equal(buffer);
```

Comment:
not your commit but I'd follow to boy scout rule and extract an assertion plugin `messageBuffer` or similar. Also consider using `sinon-chai` instead of imperatively extracting stuff out of Sinon